setup.py

```diff
@@ -3,14 +3,20 @@
     from setuptools.command.build_py import build_py as build
     from setuptools.command.build_ext import build_ext
     from setuptools.command.install import install
+    from setuptools.command.clean import clean
```

Comment:
I didn't see it here: https://github.com/pypa/setuptools/tree/master/setuptools/command but I don't know if this is a never never for any version out there.